test/fhirtypes/ElementDefinition.assignString.test.ts

```diff
@@ -1175,4 +1176,21 @@ describe('ElementDefinition', () => {
       });
     });
   });
+  describe('#checkXhtml', () => {
+    beforeEach(() => {
+      loggerSpy.reset();
+    });
+    it("should warn and throw when path doesn't start and end with div", () => {
+      const narrativeDiv = patient.findElementByPath('text.div', fisher);
+      expect(() =>
+        narrativeDiv.assignValue('<piv xmlns="http://www.w3.org/1999/xhtml">Twas brillig</div>')
+      ).toThrow();
+    });
+
+    it("shouldn't warn when path starts and ends with div", () => {
+      const narrativeDiv = patient.findElementByPath('text.div', fisher);
+      narrativeDiv.assignValue('<div xmlns="http://www.w3.org/1999/xhtml">Twas brillig</div>');
+      expect(loggerSpy.getAllLogs('info')).toHaveLength(0);
```

Comment:
This should be checking the `warn` log, not the `info` log.

Reply:
Done.